services/src/main/java/org/fao/geonet/api/processing/ValidateApi.java

```diff
@@ -170,4 +216,68 @@ public SimpleMetadataProcessingReport validateRecords(
         }
         return report;
     }
+
+
+    @ApiOperation(value = "Validate one or more records in INSPIRE validator",
+        nickname = "validateRecordsInspire",
+        notes = "Update validation status for all records.")
+    @RequestMapping(
+        value = "/validateInspire",
```

Comment:
Better to use `/validate/inspire` like in existing api ?

![image](https://user-images.githubusercontent.com/1701393/71107213-f40ab000-21c0-11ea-861d-db8df115921b.png)